tests/test_flask.py

```diff
@@ -240,7 +240,8 @@ def test_projects_search(self):
             '/projects/search/geany*', follow_redirects=True)
         self.assertEqual(output.status_code, 200)
 
-        expected = '<li class="message">Only one result matching with an '\
+        expected = '<li class="list-group-item list-group-item-default">' \
+            'Only one result matching with an '\
```

Comment:
I don't know if you care about the whitespace here (like between the `\` and the ending `'`). I noticed that it varied.